scripts/triggers-end-to-end-tests/tests.ts

```diff
@@ -187,16 +187,8 @@ describe("function triggers", () => {
       expect(test.pubsubV2TriggerCount).to.equal(1);
     });
 
-    it("should write to the scheduled pubsub emulator", async function (this) {
-      this.timeout(EMULATOR_TEST_TIMEOUT);
-
-      const response = await test.writeToScheduledPubsub();
-      expect(response.status).to.equal(200);
-      await new Promise((resolve) => setTimeout(resolve, EMULATORS_WRITE_DELAY_MS));
-    });
-
     it("should have have triggered cloud functions", () => {
-      expect(test.pubsubTriggerCount).to.equal(2);
+      expect(test.pubsubTriggerCount).to.equal(1);
```

Comment:
Although I am not completely sure, this may be because of https://github.com/firebase/firebase-tools/blob/35926f853478c959059fc97cea201d468dcb66b4/scripts/triggers-end-to-end-tests/tests.ts#L177-L183. This was always like this but some tests were removed therefore this change to `1`